modules/sitemaps/sitemap-xsl.php

```diff
@@ -90,8 +100,8 @@
 			<body>
 				<div id="description">
 					<h1>' . esc_html__( 'XML Sitemap', 'jetpack' ) . '</h1>
-					<p>' . wp_kses( sprintf( __( 'This is an XML Sitemap generated by <a href="%s" target="_blank">Jetpack</a>, meant to be consumed by search engines like <a href="%s" target="_blank">Google</a> or <a href="%s" target="_blank">Bing</a>.', 'jetpack' ), 'http://jetpack.com/', 'https://www.google.com', 'https://www.bing.com/' ), array( 'a' => array( 'href' => true, 'title' => true ) ) ) . '</p>
-					<p>' . wp_kses( sprintf( __( 'You can find more information on XML sitemaps at <a href="%s" target="_blank">sitemaps.org</a>', 'jetpack' ), 'http://sitemaps.org' ), array( 'a' => array( 'href' => true, 'title' => true ) ) ) . '</p>
+					<p>' . wp_kses( sprintf( html_entity_decode_for_xml( __( 'This is an XML Sitemap generated by <a href="%s" target="_blank">Jetpack</a>, meant to be consumed by search engines like <a href="%s" target="_blank">Google</a> or <a href="%s" target="_blank">Bing</a>.', 'jetpack' ), 'http://jetpack.com/', 'https://www.google.com', 'https://www.bing.com/' ), array( 'a' => array( 'href' => true, 'title' => true ) ) ) ) . '</p>

```

Comment:
What about using `ent2ncr` instead? It's what Core uses for RSS feeds to get valid XML. https://developer.wordpress.org/reference/functions/ent2ncr/